cal/cal/static/js/app.js

```diff
@@ -438,9 +453,10 @@ analyticsApp.controller('CalendarCtrl', function UiCalendarCtrl($scope, $http, $
 
   };
   $scope.eventRender = function(event, element, view) {
-    var location = ''
+    /* jshint unused:vars */
+    var location = '';
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7884003/21481995/566d1b14-cba9-11e6-93e7-24587b04dfad.png)
^What happens when you don't include these comments. Left is with comments included and right is without.